<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QMainWindow" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="calendarPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="buttonGrid">
          <item row="0" column="0">
           <widget class="QPushButton" name="editButton">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="quitButton">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="editPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="dateLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
